modules/docs/src/main/mdoc/docs/combinators.md

```diff
@@ -10,38 +10,38 @@ retries.
 
 ## Cheat sheet
 
-| Combinator | Context bound | Handles |
-| --- | --- | --- |
-| [`retryingOnFailures`](#retryingOnFailures) | Monad | Failures |
-| [`retryingOnSomeErrors`](#retryingonsomeerrors) | MonadError | Errors |
-| [`retryingOnAllErrors`](#retryingonallerrors) | MonadError | Errors |
-| [`retryingOnFailuresAndSomeErrors`](#retryingonfailuresandsomeerrors) | MonadError | Failures and errors |
-| [`retryingOnFailuresAndAllErrors`](#retryingonfailuresandallerrors) | MonadError | Failures and errors |
+| Combinator                                                            | Context bound | Handles             |
+| --------------------------------------------------------------------- | ------------- | ------------------- |
+| [`retryingOnFailures`](#retryingOnFailures)                           | Monad         | Failures            |
```

Comment:
A lot of this documentation will need updating, but let's leave it for now and do one big doc update at the end. We have a ticket for that.